src/components/com_tjcertificate/site/views/trainingrecord/tmpl/edit.php

```diff
@@ -95,7 +95,7 @@ class="p-5"
 
 	<div class="control-group">
 		<div class="controls">
-			<button onclick="Joomla.submitbutton('trainingrecord.save');" type="button" class="btn btn-primary"><?php echo Text::_('JSUBMIT'); ?></button>
+			<button onclick="Joomla.submitbutton('trainingrecord.apply');" type="button" class="btn btn-primary"><?php echo Text::_('JSUBMIT'); ?></button>
```

Comment:
keep it `save`. Lets redirect it back to list view @tushar33

Reply:
Done